test/util/testnode/network.go

```diff
@@ -46,7 +46,7 @@ func NewNetwork(t testing.TB, cfg *Config) (cctx Context, rpcAddr, grpcAddr stri
 	require.NoError(t, err)
 
 	t.Cleanup(func() {
-		t.Log("tearing down testnode")
+		// t.Log("tearing down testnode")
```

Comment:
What is the reason for this?

Reply:
This log is mixed when errors occur.
ex. 
```
--- FAIL: TestMaliciousTestNode (4.25s)
    /home/user/github.com/sunrise-zone/sunrise-app/test/util/malicious/app_test.go:85: 
        	Error Trace:	/home/user/github.com/sunrise-zone/sunrise-app/test/util/malicious/app_test.go:85
        	Error:      	Received unexpected error:
        	            	tx failed with code 2: tx parse error
        	Test:       	TestMaliciousTestNode
    /home/user/github.com/sunrise-zone/sunrise-app/test/util/malicious/network.go:49: tearing down testnode
   ```
   /test/util/malicious/network.go is temp file

Reply:
No log is required if successfully tearing down.